src/app/components/lineage-graph/lineage-graph.component.ts

```diff
@@ -42,11 +45,32 @@ export class LineageGraphComponent implements OnChanges, OnInit {
     public readonly DEFAULT_AVATAR_URL = AppValues.DEFAULT_AVATAR_URL;
     public readonly LineageGraphNodeKind: typeof LineageGraphNodeKind = LineageGraphNodeKind;
     public readonly DatasetKind: typeof DatasetKind = DatasetKind;
+    public INITIAL_GRAPH_VIEW_HEIGHT: number = screen.height - 400;
+    public INITIAL_GRAPH_VIEW_WIDTH: number = window.innerWidth - 120;
+    public view: [number, number] = [this.INITIAL_GRAPH_VIEW_WIDTH, this.INITIAL_GRAPH_VIEW_HEIGHT];
+    public showSidePanel = false;
+
+    @HostListener("window:resize")
+    private checkWindowSize(): void {
+        this.changeLineageGraphView();
+    }
 
     public ngOnInit(): void {
         this.graphNodes = this.nodes;
     }
 
+    public changeLineageGraphView(): void {
+        const searchResultContainer: MaybeNull<HTMLElement> = document.getElementById("searchResultContainerContent");
```

Comment:
Can it really be null?
This creates an unreachable `else` branch. 
Look for `istanbul` directives I've placed in other similar situations, it at least cuts down the impossible statements/branches from consideration.

Reply:
Use instead `ViewChild` directive